src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -298,71 +298,71 @@ public boolean isResponseCommentVisibleForUser(String userEmail, UserRole role,
         boolean isVisibleToGiver = isVisibilityFollowingFeedbackQuestion
                                  || relatedComment.isVisibleTo(FeedbackParticipantType.GIVER);
 
-        boolean userIsInstructor = role == UserRole.INSTRUCTOR;
-        boolean userIsStudent = role == UserRole.STUDENT;
+        boolean isUserInstructor = role == UserRole.INSTRUCTOR;
+        boolean isUserStudent = role == UserRole.STUDENT;
 
         boolean isVisibleToUser = isVisibleToUser(userEmail, response, relatedQuestion, relatedComment,
-                isVisibleToGiver, userIsInstructor, userIsStudent);
+                isVisibleToGiver, isUserInstructor, isUserStudent);
 
         boolean isVisibleToUserTeam = isVisibleToUserTeam(student, studentsEmailInTeam, response,
-                relatedQuestion, relatedComment, userIsStudent);
+                relatedQuestion, relatedComment, isUserStudent);
 
         return isVisibleToUser || isVisibleToUserTeam;
     }
 
     private boolean isVisibleToUserTeam(StudentAttributes student, Set<String> studentsEmailInTeam,
             FeedbackResponseAttributes response, FeedbackQuestionAttributes relatedQuestion,
-            FeedbackResponseCommentAttributes relatedComment, boolean userIsStudent) {
+            FeedbackResponseCommentAttributes relatedComment, boolean isUserStudent) {
 
-        boolean userIsInResponseRecipientTeamAndRelatedResponseCommentIsVisibleToRecipients =
-                userIsStudent
+        boolean isUserInResponseRecipientTeamAndRelatedResponseCommentIsVisibleToRecipients =
```

Comment:
One `is` should suffice: `isUserInResponseRecipientTeamAndRelatedResponseCommentVisibleToRecipients`. Similar instances below.